webapp/src/views/projects/dashboard/DashboardView.tsx

```diff
@@ -98,27 +147,27 @@ export const DashboardView = () => {
           </SecondaryBar>
         }
       >
-        <StyledContainer>
-          {statsLoadable.data && (
-            <>
-              <Box gridArea="totalStats">
-                <ProjectTotals stats={statsLoadable.data} />
-              </Box>
-              <Box gridArea="langStats">
-                <LanguageStats
-                  languageStats={statsLoadable.data.languageStats}
-                  wordCount={statsLoadable.data.baseWordsCount}
-                />
-              </Box>
-            </>
-          )}
-          <Box gridArea="activityList">
-            <ActivityList />
-          </Box>
-          <Box gridArea="activityChart">
-            <DailyActivityChart />
-          </Box>
-        </StyledContainer>
+        {anythingLoading ? (
+          <EmptyListMessage loading={true} />
```

Comment:
Base view has also loading props, doesn't it?

Reply:
Yes, but that does loading at the top

Reply:
Oki doke.